openaerostruct/geometry/geometry_mesh_transformations.py

```diff
@@ -131,11 +131,16 @@ def initialize(self):
         """
         self.options.declare("val", desc="Initial value for chord lengths")
         self.options.declare("mesh_shape", desc="Tuple containing mesh shape (nx, ny).")
+        self.options.declare(
+            "gamma",
```

Comment:
Could you pick a more intuitive name then `gamma` for the option? Maybe keeping the same as the name in the surface dictionary would make sense